typescript/sdk/src/token/adapters/CosmWasmTokenAdapter.ts

```diff
@@ -398,6 +405,5 @@ export class CwHypCollateralAdapter
     public readonly gasDenom = 'token',
   ) {
     super(chainName, multiProvider, addresses, gasDenom);
-    throw new Error('Not yet implemented');
```

Comment:
I added this constructor to throw this error. If `CwHypCollateralAdapter` works the same as `CwHypNativeAdapter` we can just delete this constructor def entirely